scripts/download-rc.bat

```diff
@@ -94,4 +94,4 @@ if defined HAZELCAST_ENTERPRISE_KEY (
 start /min "hazelcast-remote-controller" cmd /c "java -Dhazelcast.enterprise.license.key=%HAZELCAST_ENTERPRISE_KEY% -cp %CLASSPATH% com.hazelcast.remotecontroller.Main --use-simple-server> rc_stdout.txt 2>rc_stderr.txt"
 
 echo wait for Hazelcast to start ...
-ping -n 15 127.0.0.1 > nul
+ping -n 150 127.0.0.1 > nul
```

Comment:
Does this wait for 150 seconds? What is the default value for "-W" flag?

Reply:
It is 4 seconds but it is the timeout amount not the wait time for each ping. I guess the person who wrote this roughly calculated 15 pings take 10 seconds. Therefore, it should be enough to make it 150 now. Let me try it as well on local.

Reply:
![image](https://user-images.githubusercontent.com/32355782/174431726-33682a28-9999-44d9-b187-946371b607fb.png)


15 pings takes at least 15 seconds on my local which is enough. If it takes more, it is not a problem

Reply:
Thanks!